### ISSUE TITLE
Verify application

### ISSUE THREAD

**Issue by hadess** [flathub/io.github.neil_morrison44.pocket-sync#106]
For @neil-morrison44

Verification procedure:
https://docs.flathub.org/docs/for-app-authors/verification/

You should be able to easily verify the app by logging in to flathub.org with Github as the login provider, and going through the "login provider verification":
https://flathub.org/apps/manage/io.github.neil_morrison44.pocket-sync

**Comment by neil-morrison44** [comment]
👍🏻 will take a look at this tomorrow

**Comment by neil-morrison44** [comment]
That's it verified now, it does say "Failing 8 checks" though, looks like there's a couple which would (hopefully) be quick like changing the name to "Pocket Sync" & updating the screenshots. Plus some others I've been thinking about for a while like updating the App Icon which'll take a little while

---

Also I've switched that git dependency to http in v4.3.1 so hopefully that fixes the issue.
Thanks again for all the help with flathub stuff - I finally actually used it from flathub when I only had a Steam Deck handy while away.
